pilot/pkg/networking/core/v1alpha3/listener_builder_test.go

```diff
@@ -678,6 +736,7 @@ func evaluateListenerFilterPredicates(t testing.TB, predicate *listener.Listener
 }
 
 func TestSidecarInboundListenerFilters(t *testing.T) {
+	test.SetForTest(t, &features.EnableDualStack, true)
```

Comment:
Same comment as TestInboundListenerFilters()

Reply:
Hi @kfaseela, the same change as above, and I also separate them into 2 unit tests. BTW, this is the `listener` part of the unit tests, it's normal that there is no business with the IstioEndpoint instance change of supporting multiple IP addresses. That's why I add the unit tests after you ask it.

Reply:
Hello Steve, I see that you have copy pasted same tests twice, and in one of them set the feature flag.. won't it be much simpler, if the test is single, however you call it twice - once with feature flag enabled and second time without the feature flag?

Reply:
something like:

```
TestSidecarInboundListenerFilters() {
  testSidecarInboundListenerFilters(enableDualStack=false)
  testSidecarInboundListenerFilters(enableDualStack=true)
}
testSidecarInboundListenerFilters(enableDualStack)
{
  if(enableDualStack) {
      test.SetForTest(t, &features.EnableDualStack, true)
  }

  // continue with common set of tests
}
```

Reply:
done for listener_builder_test